requirements.txt

```diff
@@ -4,3 +4,5 @@ unicodecsv==0.14.1
 bz2file==0.98
 pydot==1.2.4
 requests==2.5.1
+setuptools~=57.0.0
+six~=1.16.0
```

Comment:
Do you need six at all now?

Reply:
Fix in commit [optimize requirements.txt](https://github.com/dchaplinsky/LT2OpenCorpora/pull/26/commits/3d43492ef1fcac8207b579d2cafa1146cb537f7f)